build.sbt

```diff
@@ -438,7 +456,8 @@ def assemblySettings(
     includeScala: Boolean,
     filterProvidedDeps: Boolean = true
 ): List[Def.SettingsDefinition] = {
-  // This work around need to be optional because for designer module it causes excluding of scala lib (because we has there other work around for Idea classpath and provided deps)
+  // This work around need to be optional because for designer module it causes excluding of scala lib
+  // (because we have there other work around for Idea classpath and provided deps)
```

Comment:
Is it still relevant?

Reply:
TBH I'm not sure what exact problem was solved this way and how to check if my change outdated it.

Reply:
I see that `false` was passed only for designer assembly. It looks like we have a fat jar for designer. I don't know if someone uses it :) but ... The only provided dependencies of designer where DMs so the default `filterProvidedDeps = true` should work the same so we can remove this condition